src/material.cpp

```diff
@@ -207,7 +194,7 @@ Entry* probe(const Position& pos) {
 
   // Zero or just one pawn makes it difficult to win, even with a small material
   // advantage. This catches some trivial draws like KK, KBK and KNK and gives a
-  // drawish scale factor for cases such as KRKBP and KmmKm (except for KBBKN).
```

Comment:
I don't see a problem with this comment. The code gives drawish scale factors to the cases KBBKB, KBNKB, KNNKB, KBNKN, KNNKN but not KBBKN.

Later, if you do end up removing the special endgame functions for KRKB and KRKN then it would make sense to replace the comment about KRKBP with the simpler KRKm.